core/scheduler/metrics.go

```diff
@@ -58,6 +58,13 @@ var (
 		Name:      "validator_balance_gwei",
 		Help:      "Total balance of a validator by public key",
 	}, []string{"pubkey_full", "pubkey"})
+
+	skipCounter = promauto.NewCounter(prometheus.CounterOpts{
```

Comment:
in short, how frequently are slots skipped + by how much are slots skipped. wdyt?

Reply:
Just because you "can", doesn't mean you "should". What would be the purpose? what problem do you have that this would be the answer to?  

I added this mostly to ensure the logic and scheduling and timing isn't faulty. So a simple counter is fine. I'll add a TODO to remove this if no weird behaviour is seen.

Reply:
yeah, seems like `no_of_slots` doesn't have a solid motivation. Though, my actual thought driving this was to monitor how "big" GC (or any end-of-the-world event) pauses are. But, we have logs for that so no need to add metrics i believe